src/test/scala/ai/privado/languageEngine/java/tagger/sink/DependencyTaggerTest.scala

```diff
@@ -0,0 +1,124 @@
+        Array("www.twilio.com"),
+        List(".*(com.twilio.sdk).*"),
+        false,
+        "",
+        Map(),
+        NodeType.REGULAR,
+        "",
+        CatLevelOne.SINKS,
+        "",
+        Language.JAVA,
+        Array()
+      )
+    )
+    val configAndRule = ConfigAndRules(sinks = dynamicRule)
+    val ruleCache     = RuleCache().setRule(configAndRule)
+    val twilioDep = List(
+      DependencyInfo(
+        "com.twilio.sdk",
+        "twilio",
+        "8.19.1",
+        "<artifactId>twilio</artifactId>",
+        "ThirdParties.SDK.twilio",
+        "Twilio",
+        List("www.twilio.com"),
+        List(),
+        32,
+        "pom.xml"
+      )
+    )
+    val cpg = code(
+      """
+        |<?xml version="1.0" encoding="UTF-8"?>
+        |<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        |	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+        |	<modelVersion>4.0.0</modelVersion>
+        |	<parent>
+        |		<groupId>org.springframework.boot</groupId>
+        |		<artifactId>spring-boot-starter-parent</artifactId>
+        |		<version>2.5.4</version>
+        |		<relativePath/> <!-- lookup parent from repository -->
+        |	</parent>
+        |	<groupId>com.atlan</groupId>
+        |	<artifactId>SMSService</artifactId>
+        |	<version>0.0.1-SNAPSHOT</version>
+        |	<name>SMSService</name>
+        |	<description>Demo project for Spring Boot</description>
+        |	<properties>
+        |		<java.version>1.8</java.version>
+        |		<spring-cloud.version>2020.0.3</spring-cloud.version>
+        |	</properties>
+        |	<dependencies>
+        | 		<dependency>
+        |			<groupId>mysql</groupId>
+        |			<artifactId>mysql-connector-java</artifactId>
+        |		</dependency>
+        |		<dependency>
+        |			<groupId>org.springframework.boot</groupId>
+        |			<artifactId>spring-boot-starter-web</artifactId>
+        |		</dependency>
+        |		<dependency>
+        |			<groupId>com.twilio.sdk</groupId>
+        |			<artifactId>twilio</artifactId>
+        |			<version>8.19.1</version>
+        |		</dependency>
+        |		<dependency>
+        |			<groupId>org.springframework.boot</groupId>
+        |			<artifactId>spring-boot-starter-test</artifactId>
+        |			<scope>test</scope>
+        |		</dependency>
+        |	</dependencies>
+        |	<build>
+        |		<plugins>
+        |			<plugin>
+        |				<groupId>org.springframework.boot</groupId>
+        |				<artifactId>spring-boot-maven-plugin</artifactId>
+        |			</plugin>
+        |		</plugins>
+        |	</build>
+        |</project>
+        |""".stripMargin,
+      "pom.xml"
+    ).withDependencies(twilioDep).withRuleCache(ruleCache)
+
+    "Twilio dependency should get added in processing section" in {
+      val sinkProcessing = getSinkProcessings(cpg.getPrivadoJson())
```

Comment:
Also check if we have a `sink` entry in the json, as without that. It won't be displayed on dashboard

Reply:
done